autoprotocol/protocol.py

```diff
@@ -1679,6 +1679,8 @@ def acoustic_transfer(
                 s.volume -= v
             if v > Unit(0, "microliter"):
                 transfers.append(xfer)
+            if self.propagate_properties:
+                d.add_properties(s.properties)
```

Comment:
`add_properties()` works, but I think we want to use `set_properties()` instead? In case there are duplicate keys.

Reply:
^^ I think I agree with @cj-bonilla here! this could produce a UserWarning in the stdout, which is probably fine, if not helpful to user